### ISSUE TITLE
found 16 moderate severity vulnerabilities

### ISSUE THREAD

**Issue by whoishoa** [AnomalyInnovations/serverless-bundle#80]
Hi,

Every time I add serverless-bundle to my package.json, I get 16 security vulnerabilities which I must fix manually:

When I run `npm audit fix`, it says I must fix them manually. So I run `npm audit` and I get:

`───┘
┌───────────────┬──────────────────────────────────────────────────────────────┐
│ Moderate      │ Regular Expression Denial of Service                         │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Package       │ acorn                                                        │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Patched in    │ >=7.1.1                                                      │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Dependency of │ serverless-bundle [dev]                                      │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Path          │ serverless-bundle > jest > jest-cli > jest-config >          │
│               │ jest-environment-jsdom > jsdom > acorn                       │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ More info     │ https://npmjs.com/advisories/1488 `

I tried adding installing the latest version of acorn in my package.json but I still get this issue. Does anybody know a fix?

**Comment by jayair** [comment]
Yeah there was a security update that's been added. We upgrade it for our repos.
